src/sailing_robot/tests/test_angle_average.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from sailing_robot.navigation import Navigation, angle_average
+
+class AngleAverageTests(unittest.TestCase):
+    def test(self):
+        angle_average([])
```

Comment:
These are useful as 'smoke tests' - they check that it doesn't blow up. It would be useful to have some checks of the return values. You'll probably want to use [self.assertAlmostEqual()](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual) to allow for the inevitable limited precision with floats.